README.adoc

```diff
@@ -24,11 +24,26 @@ Existing tools such as `oc` are more operations-focused and require a deep-under
 
 `odo` is designed to be simple and concise with the following key features:
 
-* Simple syntax and design centered around concepts familiar to developers, such as project, application, and component.
+* Simple syntax and design centered around concepts familiar to developers, such as projects, applications, and components.
 * Completely client based. No server is required within the OpenShift cluster for deployment.
-* Supports multiple languages and frameworks such as Node.js, Java, Ruby, Perl, PHP, and Python.
-* Detects changes to local code and deploys it to the cluster automatically, giving instant feedback to validate changes in real-time.
-* Lists all available components and services from the OpenShift cluster.
+* Official full support of Node.js and Java components.
+* Partial compatibility with languages and frameworks such as Ruby, Perl, PHP, and Python. 
+* Detects changes to local code and deploys it to the cluster automatically, giving instant feedback to validate changes in real time.
+* Lists all the available components and services from the OpenShift cluster.
+
+=== Supported languages and corresponding container images
+Run `odo catalog list components` to list the supported languages and corresponding container images present on your cluster.
+
+.Supported container images
```

Comment:
Maybe `Officially supported container images` ?

Reply:
might be a little redundant, maybe. We have "compatibility" vs "support" there now.

On a second thought, we might want to include unsupported languages there as a new table as well. But explicitly state, that no support/compatiblity is guaranteed. WDYT?